CRM/Report/Form/Contribute/Lybunt.php

```diff
@@ -315,8 +312,17 @@ public function where() {
       $this->_where .= " AND {$this->_aclWhere} ";
     }
     CRM_Financial_BAO_FinancialType::getAvailableFinancialTypes($financialTypes);
-    $this->_where .= " AND {$this->_aliases['civicrm_contribution']}.financial_type_id IN (" . implode(',' , array_keys($financialTypes)) . ")";
-    $this->_where .= " AND {$this->_aliases['civicrm_line_item']}.financial_type_id IN (" . implode(',' , array_keys($financialTypes)) . ")";
+    
+    $sql = "CREATE TEMPORARY TABLE civicrm_contribution_temp AS SELECT {$this->_aliases['civicrm_contribution']}.id {$this->_from} 
+              LEFT JOIN civicrm_line_item   {$this->_aliases['civicrm_line_item']}
```

Comment:
I think we should use INNER JOIN here, since every line item will have contribution associated with it. 

@JoeMurray : thoughts?